text/0035-verifiable-burning.md

```diff
@@ -0,0 +1,165 @@
+- Feature Name: Verifiable Burning
+- Start Date: 2022-04-07
+- MCIP PR: [mobilecoinfoundation/mcips#0035](https://github.com/mobilecoinfoundation/mcips/pull/0035)
+- Tracking Issue: https://github.com/mobilecoinfoundation/mobilecoin/issues/1779
+
+# Summary
+[summary]: #summary
+
+Create a way to provably burn funds which can be audited trustlessly by anyone.
+
+# Motivation
+[motivation]: #motivation
+
+Burning means destroying funds (or, rendering them unspendable).
+
+There are a few reasons this may be done:
+* To adjust the supply of a synthetic asset
+* To create a deflationary pressure on a token
+
+Particularly, [MCIP #25](https://github.com/mobilecoinfoundation/mcips/pull/0025)
+will provide the foundations for synthetic assets in MobileCoin.
+
+It is quite easy to make funds unspendable by just sending them to a random public address.
+If the public address is generated by hashing to curve, then it is infeasible for anyone
+to find the corresponding private keys, so no one can find and spend this money.
+
+However, in applications, we generally want burning to be transparent and auditable,
+so that anyone can easily verify when and how much funds were burned in this way.
+
+This purpose is served if the view private key of the burn address is known to everyone,
+so that they can view key scan to see the provably burned funds (and read the [MCIP #3](https://github.com/mobilecoinfoundation/mcips/pull/0003) memos
+on the burned funds, which may also be useful.)
+
+Therefore, we construct and standardize a burn address such that the view private key
+is a known constant, and the spend private keys are unknowable.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+This proposal creates a standard burn address, such that the view private key is known.
+
+To provably burn funds, simply send them to the burn address. You may attach a memo if this serves your purposes.
+
+To verify that burning occurred, use the view private key to the burn address and view-key scan the entire blockchain.
+Every provably burning event will be revealed, and you can see the block index, the amount, and read the memo field.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+The burn address construction and rationale is as follows:
+
+The `BURN_ADDRESS_VIEW_PRIVATE_KEY` is a curve25519 scalar, and is a constant known to everyone.
+
+```
+pub const BURN_ADDRESS_VIEW_PRIVATE: Scalar = Scalar::from_bits([1u8; 32]);
+```
+
+The bit pattern `[1u8; 32]` was chosen arbitrarily. It is the next simplest bit pattern to write in code after `[0u8; 32]`.
+We did not choose to make this scalar zero, because, zero is the "funniest number in cryptography" and sometimes leads to degeneracies.
+
+The `burn_address_spend_public` key is a ristretto curve point, intended to be a "nothing up my sleeve" number,
```

Comment:
small nitpick: s/ristretto curve point/ristretto element/

(Ristretto is a prime-order group, abstracted over families of potential elliptic curves, the elliptic curves themselves being implementation details. There isn't a curve equation per se to define Ristretto, as there is with a twisted Edwards curve like curve25519 or a pseudo-Koblitz curve like secp256k1.)

Reply:
thank you --

I feel like, I really want to follow the existing docu that we already have:
* All the code says `RistrettoPoint` because the type curve25519-dalek crate exposes is called `RistrettoPoint`
* Mechanics of MobileCoin also uses the term "Ristretto abstraction" rather than Ristretto curve or Ristretto group, but still calls the elements "Ristretto Point" rather than "Ristretto Element"

I'm going to try to use the term Ristretto point rather than Ristretto curve point